targets/bmv2/fuzzer.cpp

```diff
@@ -15,23 +15,31 @@ const Bmv2V1ModelProgramInfo &Bmv2V1ModelFuzzer::getProgramInfo() const {
 }
 
 InitialP4RuntimeConfig Bmv2V1ModelFuzzer::produceInitialConfig() {
+    enableInformationLogging();
```

Comment:
This is actually an option, so you do not need this.
https://github.com/p4lang/p4c/blob/main/backends/p4tools/common/p4ctool.h#L48